doc/api_changes.rst

```diff
@@ -5,6 +5,29 @@ API changes
 Here we provide information about functions or classes that have been removed,
 renamed or are deprecated (not recommended) during different release circles.
 
+DIPY 0.15 Changes
+-----------------
+
+**IO**
+
+``load_tck`` and ``save_tck`` from ``dipy.io.streamline`` has been added. They are highly recommended for managing streamlines.
+
+**Gradient Table**
+
+The default value of ``b0_thresold`` has been changed(from 0 to 50). This change can impact your algorithm.
+If you want to go back to your previous result, make sure to set up this optional argument to 0.
```

Comment:
I would write this latter as: "If you want to assure that your code runs in exactly the same manner as before, please initialize your gradient table with the keyword argument `b0_threshold` set to 0."